spec.md

```diff
@@ -1257,6 +1257,9 @@ This RPC is typically called by the CO when the workload using the volume is bei
 
 This operation MUST be idempotent.
 If the volume corresponding to the `volume_id` is not attached to the node corresponding to the `node_id`, the Plugin MUST reply `0 OK`.
+If the volume corresponding to `volume_id` or the node corresponding to
+`node_id` cannot be found by the Plugin and the volume can be safely regarded as
+ControllerUnpublished from the node, the plugin MAY return `0 OK`.
```

Comment:
Why `MAY` here? Should that be `MUST`?

Reply:
I wanted to leave room for the plugin to still return `NOT_FOUND` if it is "not sure" whether the volume is still published to the node or not. Both results should be acceptable as returning `NOT_FOUND` should result in the same behavior as currently defined (without this PR)

Reply:
OK, so the reason for not forcing SP to return `0 OK` here is for backward compatibility concern, but we still prefer SP to return `0 OK`?

In that case, i'd say we use `SHOULD`.